packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -193,6 +194,12 @@ class Autocomplete<T extends Object> extends StatelessWidget {
   /// default.
   final AutocompleteOptionsViewBuilder<T>? optionsViewBuilder;
 
+  /// Defines the maximum height the options can take
+  /// if optionsViewBuilder is null
+  ///
+  /// Defaults to 200.0
+  final double maxOptionsHeight;
```

Comment:
Suggestion:
What about `maxOptionsViewHeight`? Because it is the size of a widget, just to be more clear.

Cc @justinmc

Reply:
you mean maxOptionsViewHeight when optionsViewBuilder is assigned a custom widget? if yes
it's a custom widget user can pass  so I believe that should be constraints-free and should be left to user.

Reply:
Sorry, I meant to just improve the property name. We can leve it as-is (it was just a suggestion).